app/presenters/return-requirements/check.presenter.js

```diff
@@ -32,6 +33,21 @@ function _reasonLink (sessionId, journey) {
   return `/system/return-requirements/${sessionId}/no-returns-required`
 }
 
+function _requirements (session) {
+  const { requirements } = session
+
+  const completedRequirements = []
+
+  for (const requirement of requirements) {
+    const { siteDescription, agreementsExceptions } = requirement
+    if (agreementsExceptions) {
```

Comment:
Adding some white space

```suggestion
    const { siteDescription, agreementsExceptions } = requirement

    if (agreementsExceptions) {
```